rotkehlchen/api/v1/schemas.py

```diff
@@ -726,6 +726,10 @@ def make_ledger_action(  # pylint: disable=no-self-use
         return {'action': LedgerAction(**data)}
 
 
+class LedgerActionsDeleteSchema(Schema):
+    identifiers = DelimitedOrNormalList(fields.Integer(required=True), required=True)
+
+
 class IntegerIdentifierSchema(Schema):
```

Comment:
The same way we have `IntegerIdentierSchema` whouldn't it make sense to call it `IntegerIdentifierListSchema`? or something on the line avoiding the usage of LedgerAction